src/Telemetry/AzureDevOpsTimeline/AzureDataTypes.cs

```diff
@@ -305,4 +305,10 @@ public sealed class ArtifactResource
         /// </summary>
         public string Url { get; set; }
     }
+
+    public class Goals
+    {
+        public int Definition_Id { get; set; }
+        public int Total_minutes { get; set; }
```

Comment:
Inconsistent formatting. This should either be

```
Definition_Id
Total_Minutes
```

or 

```
DefinitionId
TotalMinutes
```

Reply:
I'd prefer it be the latter, with a `[JsonProperty("total_minutes")]` on it to help serialize.